sample/src/debug/java/com/barista_v/sample/MyActivity.java

```diff
@@ -26,11 +31,16 @@ public void setContentView(@LayoutRes int layoutResID) {
 
     String[] hosts = new String[] { "Value 1", "Value 2" };
 
+    BugRepository.Builder repositoryBuilder =
+        new BugReportRepositoryBuilder("8d4f9d99c2a4c818746fe1cb9015e2c9", "1954523",
```

Comment:
any chance we don't expose the apikey and projectid in the codebase? maybe some configuration file?